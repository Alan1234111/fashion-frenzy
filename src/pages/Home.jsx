import "bootstrap/dist/css/bootstrap.min.css";
import Button from "react-bootstrap/Button";
import {Link} from "react-router-dom";
import Container from "react-bootstrap/esm/Container";
import Row from "react-bootstrap/esm/Row";
import Col from "react-bootstrap/esm/Col";

function Home() {
  return (
    <Container fluid id="home" className="d-flex align-items-center">
      <Row className="d-flex justify-content-center ">
        <Col xs={12} md={6} className="d-flex flex-column align-items-center p-5 ">
          <h1 className="text-center text-white fw-bold">Welcome to FashionFrenzy!</h1>
          <h2 className="text-white fs-6 text-center">Discover FashionFrenzy, where style meets passion. Explore our meticulously curated collection, blending timeless classics with cutting-edge designs. Embrace your unique fashion sense with our diverse range of clothing, accessories, and footwear. Step into a fashion sanctuary and unleash your style like never before.</h2>
          <Button as={Link} to="/products" variant="danger" className="mt-4">
            SHOP NOW
          </Button>
        </Col>
      </Row>
    </Container>
  );
}

export default Home;
