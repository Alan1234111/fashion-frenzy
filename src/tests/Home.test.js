import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import Home from "../pages/Home";

describe("Home", () => {
  it("matches the snapshot", () => {
    const tree = renderer
      .create(
        <MemoryRouter initialEntries={["/"]} initialIndex={0}>
          <Home />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
