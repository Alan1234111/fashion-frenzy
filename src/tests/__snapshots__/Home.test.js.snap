// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<div
  className="d-flex align-items-center container-fluid"
  id="home"
>
  <div
    className="d-flex justify-content-center  row"
  >
    <div
      className="d-flex flex-column align-items-center p-5  col-md-6 col-12"
    >
      <h1
        className="text-center text-white fw-bold"
      >
        Welcome to FashionFrenzy!
      </h1>
      <h2
        className="text-white fs-6 text-center"
      >
        Discover FashionFrenzy, where style meets passion. Explore our meticulously curated collection, blending timeless classics with cutting-edge designs. Embrace your unique fashion sense with our diverse range of clothing, accessories, and footwear. Step into a fashion sanctuary and unleash your style like never before.
      </h2>
      <a
        className="mt-4 btn btn-danger"
        disabled={false}
        href="/products"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        SHOP NOW
      </a>
    </div>
  </div>
</div>
`;
